modules/mobile/app/models/mobile/v0/adapters/immunizations.rb

```diff
@@ -6,31 +6,39 @@ module Adapters
       class Immunizations
         def parse(immunizations)
           vaccines = vaccines(immunizations)
-
+          date_text_list = []
```

Comment:
I think this could be done a bit more simply without creating the date_text_list. 
```
immunizations[:entry].each_with_object([]) do |i, acc|
  cvx_code = cvx_code(immunization[:vaccine_code])
  vaccine = vaccines&.find_by(cvx_code: cvx_code)
  date = date(immunization)
  text = immunization[:vaccine_code][:text]

  next if acc.any? {|immunization| immunization.date == date && immunization.short_description == text }
  
  # create immunization record
  
  acc << immunization
end
```

Reply:
Interesting, yeah, that's a good idea. I'll give that a try, or possibly something simpler

Reply:
End up using array.uniq. Seemed like the simplest solution, must have looked over it when I was looking at the array methods.